packages/vm/src/instance.rs

```diff
@@ -85,7 +86,12 @@ where
     ) -> VmResult<Self> {
         let store = module.store();
 
-        let env = Environment::new(backend.api, gas_limit, print_debug);
+        let env = Environment::new(backend.api, gas_limit);
+        if print_debug {
+            env.set_debug_handler(Some(Rc::new(|msg: &str, gas_remaining| {
+                println!("{msg} (gas: {gas_remaining})");
+            })))
+        }
```

Comment:
Here we define the default implementation. This was just `println!("{}", msg)` before and now contains extra information. Let's talk about which default we want. Should we print to STDERR instead? I'd say so. Do we want to have high precision timing by default, no so sure.

Reply:
The default implementation is back to the simple form as before, but writes to STDERR. A new integration test in cyberpunk shows gas and runtime logging.